Traceback (most recent call last):
  File "fuzz.py", line 37, in <module>
    fuzz()
  File "fuzz.py", line 9, in fuzz
    with zipfile.ZipFile(file=fp, mode="r",) as zp:
  File "zipfile.py", line 1200, in __init__
    self._RealGetContents()
  File "zipfile.py", line 1302, in _RealGetContents
    filename = filename.decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 17: invalid continuation byte
